docs/parsers/CanvasPathGeometry.md

```diff
@@ -0,0 +1,223 @@
+```
+
+### Creating CanvasStrokeStyle
+
+The following API converts a style data string to **CanvasStrokeStyle** object
+
+```cs
+public static CanvasStrokeStyle CreateStrokeStyle(string styleData);
+```
+
+Example
+
+```cs
+// CanvasStrokeStyle with custom dash style
+var customStrokeStyle = CanvasPathGeometry.CreateStrokeStyle(“CSS CDS 2 2 0 2 1 3”);
+```
+
+### Creating ICanvasStroke
+
+**ICanvasStroke** is an interface to represent the Stroke which can be used to render an outline on a **CanvasGeometry**. **CanvasStroke** class implements this interface.
+
+```cs
+public interface ICanvasStroke
+{
+    ICanvasBrush Brush { get; }
+    float Width { get; }
+    CanvasStrokeStyle Style { get; }
+    Matrix3x2 Transform { get; set; }
+}
+
+public sealed class CanvasStroke : ICanvasStroke
+{
+    public ICanvasBrush Brush { get; }
+    public float Width { get; }
+    public CanvasStrokeStyle Style { get; }
+    public Matrix3x2 Transform { get; set; }
+
+    public CanvasStroke(ICanvasBrush brush, float strokeWidth = 1f);
+    public CanvasStroke(ICanvasBrush brush, float strokeWidth, CanvasStrokeStyle strokeStyle);
+    public CanvasStroke(ICanvasResourceCreator device, Color strokeColor, float strokeWidth = 1f);
+    public CanvasStroke(ICanvasResourceCreator device, Color strokeColor, float strokeWidth, CanvasStrokeStyle strokeStyle);
+}
+```
+
+The following API converts a stroke data string to **ICanvasStroke** object
+
+```cs
+public static ICanvasStroke CreateStroke(ICanvasResourceCreator resourceCreator, string strokeData);
+```
+
+Example
+
+```cs
+// A dashed stroke rendered with a LinearGradientBrush and a custom stroke style
+var dashedStroke = CanvasPathGeometry.CreateStroke(device, “ST 4.5 LG M 0 0 Z80 80 S 0.00 #ffff0000, 0.5 #ff00ff00, 0.99 #ff0000ff CSS DS 2 Do 2 SC 1 EC 2 CDS 2 2 0 2 1 3”);
+// A solid line stroke of thickness 2 having red color
+var solidStroke = CanvasPathGeometry.CreateStroke(device, “ST 2 SC #ff0000”);
+```
+
+### Compositor Extension Methods
+
+Based on the above methods, the following extension methods for **Compositor** are now available
+
+```cs
+public static CompositionPath CreatePath(this Compositor compositor, string pathData);
+public static CompositionPathGeometry CreatePathGeometry(this Compositor compositor, string pathData);
+public static CompositionSpriteShape CreateSpriteShape(this Compositor compositor, string pathData);
+public static CompositionGeometricClip CreateGeometricClip(this Compositor compositor, CanvasGeometry geometry);
+public static CompositionGeometricClip CreateGeometricClip(this Compositor compositor, string pathData);
+```
+
+### Creating a Squircle
+
+A [Squircle](https://en.wikipedia.org/wiki/Squircle) is an intermediate shape between a square and a circle.
+
+<img src="https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/parsers/Squircle.gif" />
```

Comment:
You should put the image in the resources folder and reference it with a relative path within a Markdown style image link:

`![Use Case 2 Output](../resources/images/Animations/Chaining-Animations-Blur-Fade-Rotate.gif)`

Not sure if we can have a smaller file though 3MB is pretty large.

Reply:
At 2266x975 this also seems much bigger than it needs to be when displayed on the docs site.

Reply:
Resized image to 960px x 413px. Filesize reduced to 1MB.
Moved image to ../resources/images folder.